### ISSUE TITLE
Build failure: modrinth-app

### ISSUE THREAD

**Issue by MrSom3body** [NixOS/nixpkgs#335450]
### Steps To Reproduce

1. build *modrinth-app*

### Build log

https://gist.github.com/MrSom3body/6e5290ec32286de2e2ffba6188c1c74a

### Additional context

It worked on the 13th August 2024, but I didn't try it after that, so I can't really tell since when it stopped building.

### Notify maintainers

@getchoo 

### Metadata

```console
[user@system:~]$ nix-shell -p nix-info --run "nix-info -m"
- system: `"x86_64-linux"`
 - host os: `Linux 6.10.4-xanmod1, NixOS, 24.11 (Vicuna), 24.11.20240814.c3aa7b8`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.18.5`
 - channels(root): `"home-manager, nixos-24.05"`
 - nixpkgs: `/nix/store/sfycwi72zfjsspidinx56ajaiffpyh17-source`
```

---

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

**Comment by getchoo** [comment]
this is going to be due to https://github.com/NixOS/nixpkgs/issues/332957. forgot to check this manually since hydra can't build it :(

**Comment by getchoo** [comment]
Looking at the log on desktop now, this isn't actually a `modrinth-app` build failure, but rather one in our dependencies

> error: builder for '/nix/store/f1x7w9gf00cvbz2v64wd20hm7yx27c4c-tauri-1.7.1.drv' failed with exit code 101;

This was fixed in #334771, however that PR seems to have *actually* broken the `modrinth-app` package due to it bumping the dependency to an unsupported version. #335334 should fix all of this mess (fingers crossed)
